core/src/main/java/io/confluent/kafka/schemaregistry/metrics/SchemaRegistryMetric.java

```diff
@@ -16,31 +16,50 @@
 package io.confluent.kafka.schemaregistry.metrics;
 
 import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.MeasurableStat;
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.Sensor;
 import org.apache.kafka.common.metrics.stats.Value;
 
 import java.util.concurrent.atomic.AtomicLong;
 
 public class SchemaRegistryMetric {
+  @Deprecated
```

Comment:
Do we need these deprecated fields anymore or do we plan on cleaning them up in a separate PR ?